src/NetAnalyzers/UnitTests/Microsoft.CodeQuality.Analyzers/QualityGuidelines/MarkMembersAsStaticTests.cs

```diff
@@ -1016,9 +1016,18 @@ public class C : System.Web.HttpApplication
     // The following controller methods can't be made static
     protected void Application_Start() { }
     protected void Application_End() { }
-}");
+}";
 
-            await VerifyVB.VerifyAnalyzerAsync(@"
+            const string editorConfigText = "build_property.UsingMicrosoftNETSdkWeb = true";
+            var csTest = new VerifyCS.Test()
```

Comment:
Any reason you have split the test and the `RunAsync`? You could write something like

```csharp
await new VerifyCS.Test
{
	TestCode = csSource,
	AnalyzerConfigDocument = editorConfigText,
}.RunAsync();
```

Reply:
In case you apply this comment, note that it applies to the various tests you have changed.

Reply:
Awaiting an object creation with initializer with an async method invocation seems too chatty to me :-). I prefer to keep object creation with initializer as its own statement.

Reply:
Seems like other tests in this file follow the pattern you suggested. I will switch the pattern in new tests to make it consistent.